.coafile

```diff
@@ -12,12 +12,22 @@ use_spaces = True
 # will be executed sequentially; also we need the LineLengthBear to double
```

Comment:
That isnt particularly helpful either.

`.coafile: Activate remove_all_unused_imports`

?

btw, this placement of the commit message review will cause it to stick around. ala https://github.com/coala/coala/issues/4087
However, it does allow it to be replied to.
GitHub sucks for reviews.